vaadin-date-time-picker-flow/pom.xml

```
@@ -142,12 +142,12 @@
         <dependency>
             <groupId>org.webjars.bowergithub.polymerelements</groupId>
             <artifactId>iron-scroll-target-behavior</artifactId>
-            <version>2.1.0</version>
+            <version>2.1.1</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-item</artifactId>
-            <version>2.1.1</version>
+            <version>2.2.0-alpha1</version>
```

Change to stable version
```suggestion
            <version>2.2.0</version>
```